core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -43,6 +43,21 @@ $.extend( $.fn.dataTable.ext.type.order, {
         a = ConvertDurationString( a );
         b = ConvertDurationString( b );
         return ((a < b) ? 1 : ((a > b) ? -1 : 0));
+    },
+
+    
+    "size-pre": parseFloat,
+
+    "size-asc": function ( a, b ) {
+        a = parseFloat( a );
+        b = parseFloat( b );
+        return ((a < b) ? -1 : ((a > b) ? 1 : 0));
+    },
+
+    "size-desc": function ( a, b ) {
+        a = parseFloat( a );
+        b = parseFloat( b );
```

Comment:
Should we care the case if a or b is `NaN`?
If the value of a metrics is 0, `parseFloat("")` will be performed and it returns `NaN` right?